xtask/src/lib.rs

```diff
@@ -144,10 +144,10 @@ pub fn build_documentation(
 
     let docs_path = windows_safe_path(
         &workspace
-            .join(package.to_string())
             .join("target")
             .join(target)
-            .join("doc"),
+            .join("doc")
+            .join(package.to_string().replace('-', "_")),
```

Comment:
This change means we can't generate documentation for the packages excluded from the workspace. Not sure if it matters, I don't think we want to do that for the xtensa-lx packages or hil tests at this moment.